lib/scope_delegation.rb

```diff
@@ -5,9 +5,14 @@ def delegate_scope(name, options)
     source      = options[:scope] || name
     association = reflect_on_association(options[:to]) || raise("Unknown association")
     name        = [options[:prefix], name].compact.join("_")
+    grouping    = options[:group]
 
     scope name, Proc.new { |*args|
-      joins(association.name).merge(association.klass.send(source, *args))
+      if grouping
+        joins(association.name).merge(association.klass.send(source, *args)).group(grouping)
+      else
+        joins(association.name).merge(association.klass.send(source, *args))
+      end
```

Comment:
tried it and it broke things...i can dive in a little more next FF...and add specs :)